docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -59,4 +59,35 @@ For example:
 
 ----
 minishift start --iso-url centos
-----
\ No newline at end of file
+----
+
+[[x509-certificate-is-valid-for-10.0.2.15-127.0.0.1-192.168.99.100-not-192.168.99.101]]
+== x509 certificate is valid for 10.0.2.15, 127.0.0.1, 172.17.0.1, 172.30.0.1, 192.168.99.100, not 192.168.99.101
+
+If you started a stopped {project} VM then due to IP change you might hit following x509 certificate error:
+
+----
+$ minishift.exe start
+-- Checking if requested hypervisor 'kvm' is supported on this platform ... OK
+-- Checking the ISO URL ... OK
+-- Starting local OpenShift cluster using 'kvm' hypervisor ...
+-- Starting Minishift VM ......................... OK
+[...]
+FAIL
+   Error: cannot access master readiness URL https://192.168.99.101:8443/healthz/ready
+   Details:
+     No log available from "origin" container
+
+   Caused By:
+     Error: Get https://192.168.99.101:8443/healthz/ready: x509: certificate is valid for 10.0.2.15, 127.0.0.1, 172.17.0.1, 172.30.0.1, 192.168.99.100, not 192.168.99.101
+Error during 'cluster up' execution: Error starting the cluster.
+----
+
+The reason is openshift cluster certificates contain IP when {project} VM created and a restart may change the VM IP for which certificates are not valid anymore.
```

Comment:
The reason for the above error is  that OpenShift cluster certificates contain the IP  of the {project} VM. The certificates are generated only when {project} VM is freshly started (not for restart). So when a restart changes the VM IP the certificates are not valid anymore.

Reply:
> The certificates are generated only when {project} VM is freshly started (not restart).

@LalatenduMohanty By "freshly started (not restart)", I assume that the certificates are generated the first time that `minishift start` is run, not for any following runs of `minishift start`, correct? If that's the case, I'd like to be explicit about that.

Reply:
@robin-owen yes, thats right.